client/src/components/common/ContentPage.tsx

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Container } from 'react-bootstrap';
 
-/**
- * @typedef {object} ContentPageProps
- * @property {string}           title     Page Title
- * @property {React.ReactNode}  children  Children
- */
+interface ContentPageProps {
+  /** Title of page */
+  title: string,
+  /** React children to be passed through */
+  children: React.ReactNode,
+}
 
 /**
  * Content Page container component
  *
  * @param {ContentPageProps} props Props
  * @returns {React.Component<ContentPageProps>} Component
```

Comment:
We declare the types using TS. Also might be confusing if the JSDoc types are different from the TS declarations.